es:
  devise:
    omniauth_callbacks:
      user:
        failure: "No se pudo autenticar desde %{kind} debido a \"%{reason}\"."
        success: "AutenticaciÃ³n satisfactoria desde tu cuenta de %{kind}."
